tests/backward/src/test/java/org/apache/bookkeeper/tests/backward/TestBackwardCompat.java

```diff
@@ -314,22 +318,34 @@ void stop() throws Exception {
     }
 
     static class LedgerCurrent {
```

Comment:
please bring `4.6` server into testing scope also. you need to test the compatibility between 4.7 and 4.6/4.5.

After 4.6 is out, we need to enforce EOL to clean up backward compat tests.

Reply:
Yes but I think we need a separate patch to refactor backward compatibility tests.
And with Jia and Ivan today we saw that it is not possible to run the request kind of tests with the actual framework,please see the conversation on slack

Reply:
yes, I read the conversation. for this change to be checked in, please test backward between 4.7 and 4.6/4.5. if there is a need for refactor backward compatibility tests, lets refactor them now rather than later. that says I can image more and more changes checked in, we should have something to guarantee we don't break backward compatibility. that says I would mark backward test between 4.7 and 4.6/4.5 as a blocker for this pull request.